packages/hydrated_bloc/lib/src/hydrated_bloc.dart

```diff
@@ -366,8 +366,8 @@ mixin HydratedMixin<State> on BlocBase<State> {
   /// Storage prefix which can be overridden to provide a custom
   /// storage namespace.
   /// Defaults to [runtimeType] but should be overridden in cases
-  /// where stored data should be resilient to obfuscation or persist
-  /// between debug/release builds.
+  /// where stored data should be resilient to obfuscation, minification (web)
```

Comment:
```suggestion
  /// where stored data should be resilient to obfuscation, minification
```

Nit but I don't think (web) adds much here so I'd probably just remove it especially since we specifically call out web in the README changes above.